source/system/Character.ooc

```diff
@@ -6,7 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-include stdlib
+include stdlib, stdint
```

Comment:
This is to have access to `uint64_t` on windows, i think this header is included by something else on unix, and on windows you need to do that manually.